website/components/learn-callout/style.css

```diff
@@ -66,12 +66,14 @@
     }
 
     & > a {
+      display: flex;
```

Comment:
This fixes an issue where the styles for this element existed in multiple places.